openmrs/src/main/java/org/motechproject/openmrs/resource/impl/ProgramEnrollmentResourceImpl.java

```diff
@@ -81,7 +81,7 @@ private Gson buildGsonWithAdapters() {
                 .registerTypeAdapter(Program.class, new Program.ProgramSerializer())
                 .registerTypeAdapter(Program.State.class, new Program.State.ProgramSerializer())
                 .registerTypeAdapter(Location.class, new Location.LocationSerializer())
-                .registerTypeAdapter(Attribute.class, new Attribute.BahmniProgramEnrollmentAttributeSerializer())
```

Comment:
Since Bahmni's OpenMRS implementation supports sending attributes with both:
- attributeType as an object with "uuid" property
and
- attributeType as a property with uuid value

I removed extra serializer